src/IssueViz.Security/UI/HotspotsList/HotspotsControl.xaml

```diff
@@ -4,223 +4,61 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:core="clr-namespace:SonarLint.VisualStudio.Core.WPF;assembly=SonarLint.VisualStudio.Core"
              xmlns:issueVisualizationControl="clr-namespace:SonarLint.VisualStudio.IssueVisualization.IssueVisualizationControl;assembly=SonarLint.VisualStudio.IssueVisualization"
-             xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0"
              xmlns:vsCatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
              xmlns:vsImaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
              xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
              xmlns:hotspotsList="clr-namespace:SonarLint.VisualStudio.IssueVisualization.Security.UI.HotspotsList"
              DataContext="{Binding ViewModel, RelativeSource={RelativeSource Mode=Self}}">
     <UserControl.Resources>
-        <vsTheming:BrushToColorConverter x:Key="BrushToColorConverter"/>
```

Comment:
The diff of this file is a bit ugly.
Overview:
* added a new `ResourceDictionary` section and merged the shared resources into it.
* the remaining hotspot-specific resources appear below the merged dictionary
* any shareable resources were moved to the new resources file.

The only style that needed to be edited was `DataGridRow`. I'll comment it separately.